src/components/Tooltip/README.md

```diff
@@ -98,6 +98,34 @@ Use when the tooltip overlays interactive elements when active, for example a fo
 </div>
 ```
 
+### Tooltip as TextField helper
+
+Use only when necessary to provide an explanation for a non-trivial user input.
+
+```jsx
+function TooltipTextFieldExample() {
+  const [value, setValue] = useState('');
+
+  const handleChange = useCallback((newValue) => setValue(newValue), []);
+
+  return (
+    <div style={{padding: '75px 0', width: '300px'}}>
+      <TextField
+        id="atPrice"
+        value={value}
+        label="Compare at price"
+        onChange={handleChange}
+        suffix={
+          <Tooltip content="To show a reduced price, move the variant's original price into Compare at price. Enter a lower value into Price.">
+            <Icon source={QuestionMarkMinor} />
+          </Tooltip>
+        }
+      />
+    </div>
+  );
+}
+```
+
```

Comment:
We normally don't add examples for one off bugs. Are you able to remove this and the additional icon?

We do have a part of the PR template where you can paste a `playground` this code would be excellent to add there ✨